src/main/resources/db/migration/V3__SUBSUMSJON_BRUKT.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS v1_subsumsjon_brukt
+(
+    id VARCHAR(64) NOT NULL,
```

Comment:
Burde vi ha en foreign key til v1_subsumsjon ala

`FOREIGN KEY (id) REFERENCES V1_SUBSUMSJON (id)` 

sånn for konsistens

Reply:
Kanskje det, skal vi sette opp noen cascades?